src/Catalyst.Core.Lib.Tests/UnitTests/IO/Events/TransactionReceivedEventTests.cs

```diff
@@ -49,10 +49,11 @@ public sealed class TransactionReceivedEventTests
         private readonly IBroadcastManager _broadcastManager;
         private readonly TransactionReceivedEvent _transactionReceivedEvent;
         private readonly IPeerSettings _peerSettings;
+        private readonly TestMapperProvider _mapperProvider;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field '_mapperProvider' and declare it as a local variable in the relevant methods.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4404007)